components/items/itemPage/CommentThread.tsx

```diff
@@ -0,0 +1,147 @@
+
+const CommentContainer = styled.div`
+  padding: 24px 0;
+  position: relative;
+`;
+
+const SeeMoreButton = styled.button`
+  position: absolute;
+  right: 0;
+`;
+
+const CommentContent = styled.p`
+  font-size: 16px;
+  line-height: 140%;
+  margin-bottom: 24px;
+`;
+
+const AuthorProfile = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 8px;
+`;
+
+const UserProfileImage = styled.img`
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+  object-fit: cover;
+`;
+
+const Username = styled.p`
+  color: var(--gray-600);
+  font-size: 14px;
+  margin-bottom: 4px;
+`;
+
+const Timestamp = styled.p`
+  color: ${({ theme }) => theme.colors.gray[400]};
+  font-size: 12px;
+`;
+
+interface CommentItemProps {
+  item: ProductComment;
+}
+
+const CommentItem: React.FC<CommentItemProps> = ({ item }) => {
+  const authorInfo = item.writer;
+  const formattedTimestamp = formatUpdatedAt(item.updatedAt);
+
+  return (
+    <>
+      <CommentContainer>
+        {/* 참고: 더보기 버튼 기능은 추후 요구사항에 따라 추가 예정 */}
+        <SeeMoreButton>
+          <SeeMoreIcon />
+        </SeeMoreButton>
+
+        <CommentContent>{item.content}</CommentContent>
+
+        <AuthorProfile>
+          <UserProfileImage
+            src={authorInfo.image || DefaultProfileImage}
+            alt={`${authorInfo.nickname}님의 프로필 사진`}
+          />
+
+          <div>
+            <Username>{authorInfo.nickname}</Username>
+            <Timestamp>{formattedTimestamp}</Timestamp>
+          </div>
+        </AuthorProfile>
+      </CommentContainer>
+
+      <LineDivider $margin="0" />
+    </>
+  );
+};
+
+const ThreadContainer = styled.div`
+  margin-bottom: 40px;
+`;
+
+interface CommentThreadProps {
+  productId: number;
+}
+
+const CommentThread: React.FC<CommentThreadProps> = ({ productId }) => {
+  const [comments, setComments] = useState<ProductComment[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState<string | null>(null);
+
+  useEffect(() => {
+    if (!productId) return;
+
+    const fetchComments = async () => {
+      setIsLoading(true);
+
+      try {
+        const response: ProductCommentListResponse = await getProductComments({
+          productId,
+        });
+        setComments(response.list);
+        setError(null);
+      } catch (error) {
+        console.error("Error fetching comments:", error);
+        setError("상품의 댓글을 불러오지 못했어요.");
+      } finally {
+        setIsLoading(false);
+      }
+    };
+
+    fetchComments();
+  }, [productId]);
+
+  if (isLoading) {
+    return <div>상품 댓글 로딩중...</div>;
+  }
+
+  if (error) {
+    return <div>오류: {error}</div>;
+  }
+
+  if (comments && !comments.length) {
+    return <EmptyState text="아직 문의가 없습니다." />;
+  } else {
+    return (
+      <ThreadContainer>
+        {comments.map((item) => (
+          <CommentItem item={item} key={`comment-${item.id}`} />
+        ))}
+      </ThreadContainer>
+    );
+  }
```

Comment:
P2:
이미 comments && !comments.length 조건문에서 return을 하고 있으니 else문은 사용하지 않으셔도 됩니다.
```suggestion
  if (comments && !comments.length) {
    return (
      <EmptyState text={`아직 댓글이 없어요,\n지금 댓글을 달아 보세요!`} />
    );
  } 
  return (
    <ThreadContainer>
      {comments.map((item) => (
        <CommentItem item={item} key={`comment-${item.id}`} />
      ))}
    </ThreadContainer>
  );
};
```